includes/admin/class-sensei-onboarding.php

```diff
@@ -126,6 +129,91 @@ public function onboarding_page() {
 		<?php
 	}
 
+	/**
+	 * Check if current screen is selected to display the wizard notice.
+	 *
+	 * @return boolean
+	 */
+	private function is_current_screen_selected_to_wizard_notice() {
```

Comment:
To clarify, maybe a better name might be `current_page_should_display_onboarding_wizard` or something like that? The idea of it being "selected" might be misleading.

Reply:
I think functions should start with a verb where possible (i.e. `should_current_page_display_setup_wizard`).

Reply:
Looks better! https://github.com/Automattic/sensei/pull/3096/commits/6be26d2ef7e0cb751426d5a0a4d62c6b48ef3b47 "Selected" wasn't really a good name. 😅